third_party/maya/lib/pxrUsdMayaGL/CMakeLists.txt

```diff
@@ -29,6 +29,9 @@ pxr_shared_library(${PXR_PACKAGE}
     PUBLIC_CLASSES
         batchRenderer
         hdRenderer
+		usdTaskDelegate
+        usdShapeRenderer
+        usdBatchRenderer
```

Comment:
`UsdBatchRenderer` looks like it may have begun life as a copy of `UsdMayaGLBatchRenderer` and then has all of your performance improvements and refactoring added. Does it make sense for both to continue to exist? Should `UsdMayaGLBatchRenderer` be deprecated/removed now in favor of `UsdBatchRenderer`?